mne/cov.py

```diff
@@ -1125,9 +1129,9 @@ def prepare_noise_cov(noise_cov, info, ch_names, rank=None,
     else:
         C = np.diag(noise_cov.data[C_ch_idx])
 
-    scalings_ = dict(mag=1e12, grad=1e11, eeg=1e5)
+    _scalings = DEFAULTS['scalings_cov_rank']
     if isinstance(scalings, dict):
-        scalings_.update(scalings)
+        _scalings.update(scalings)
```

Comment:
Arfff. Now that we have the global module level variable this is actually not any longer safe code I fear. I think we need to do a copy here. 

```_scalings = dict(DEFAULTS['scalings_cov_rank'])```